mounts_test.go

```diff
@@ -49,6 +50,22 @@ func TestMounts(t *testing.T) {
 			}`),
 		},
 
+		//file mount from raw
+		&FileMount{
+			File: filepath.Join("preloaded", "raw.txt"),
+			Content: json.RawMessage(`{
+				"raw": "Hello Raw Content!"
+			}`),
+		},
+
+		//file mount from base64
+		&FileMount{
+			File: filepath.Join("preloaded", "base64"),
+			Content: json.RawMessage(`{
+				"base64": "ZWNobyAiSGVsbG8gQmFzZTY0ISI="
```

Comment:
Technically, I think this should be `ZWNobyAnSGVsbG8gQmFzZTY0IScK` since `!` is not escaped by double quotes in bash, but it doesn't really matter. 😉